.github/workflows/cd.yml

```diff
@@ -31,12 +31,30 @@ jobs:
           ruby-version: 3.3.5
           bundler-cache: true
 
+      - name: Set up Unlighthouse Scan
+        run: npm install -g @unlighthouse/cli puppeteer
```

Comment:
this report generation should be outside deploy action, with the same conditions, but that's another thing tather deploy